assets/src/application/ui/display/icons/HelpIcon/index.tsx

```diff
@@ -8,24 +8,18 @@ import { IconProps } from '../types';
 import './style.scss';
 
 interface HelpIconProps extends IconProps {
-    clickable?: boolean;
-    tooltipText?: string;
+	clickable?: boolean;
+	tooltipText?: string;
 }
 
 const HelpIcon: React.FC<HelpIconProps> = ({ clickable, tooltipText, ...props }) => {
-    const className = classNames('ee-help-icon', props.className);
+	const className = classNames('ee-help-icon', props.className);
 
```

Comment:
Is indentations messed up here?

Reply:
yes, I had some issues with VS Code few months ago and ended up reinstalling it fully